lib/commands/update.js

```diff
@@ -179,6 +181,25 @@ class UpdateCommand extends Command {
         });
     }
 
+    rollbackFromFail(error, argv) {
+        const previousVersion = this.system.getInstance().cliConfig.get('previous-version');
+        this.ui.error(error);
+        const question = argv.version === '2.0.0' ?
+            `Unable to upgrade to Ghost 2.0 - Would you like to revert back to ${previousVersion}?` :
+            `Unable to upgrade Ghost - Would you like to revert back to ${previousVersion}?`;
+        const promise = argv['auto-rollback'] ? Promise.resolve(true) : this.ui.confirm(question, true);
+
+        return promise.then(answer => {
+            if (!answer) {
+                return Promise.resolve();
+            }
+
+            return this.run({
+                rollback: true
```

Comment:
We can ensure a database rollback with https://github.com/TryGhost/knex-migrator/pull/110 and https://github.com/TryGhost/Ghost-CLI/pull/774 👍

Reply:
Will review this PR when we have merged #774 :)